style.css

```diff
@@ -320,6 +344,88 @@ textarea:focus {
   background-color: #60c095;
 }
 
+/* modal section */
+.project-card-bg {
+  width: 100%;
+  height: 100vh;
+  background: #c1c7d0;
+  padding: 20px;
+  display: grid;
+  position: fixed;
+  overscroll-behavior: contain;
+  overflow: auto;
+  z-index: 100;
+  left: 0;
+  top: 0;
+  animation: ease-in-out 1s;
+}
+
+.project-card {
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: white;
+  align-self: center;
+  position: relative;
+  min-height: 855px;
+  margin: auto;
+}
+
+.close-project {
+  font-size: 25px;
+  color: #67798e;
+  position: absolute;
+  left: 87%;
+  top: 4%;
+  background: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.project-thumbnail img {
+  width: 100%;
+  height: 100%;
+}
+
+.project-card-heading {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.flex-list {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  grid-gap: 10px;
+  list-style: none;
+  align-items: center;
+  text-align: center;
+  margin: 16px 0 29px 0;
+  white-space: nowrap;
+}
+
+.project-buttons {
+  display: flex;
+  gap: 12px;
+  justify-content: center;
```

Comment:
You can remove `justify-content: center` to align the buttons to the left. To match the design.